count.py

```diff
@@ -40,10 +52,20 @@ def count(file, column, limit, interval, match, update_minutes, outfile):
                     dtime = datetime.strptime(row[indices[1]][0:-9], '%Y/%m/%d %H:%M:%S')
                     if dtime.hour < int(interval[0]) or dtime.hour > int(interval[1]):
                         continue
-                count_map[row[index]] = count_map.get(row[index], 0) + 1
 
-        alert_list = sorted(count_map.items(),
-                            key = lambda kv:(kv[1], kv[0]), 
+                current_count = count_map.get(row[index], 0)
+                if current_count != 0:
+                    current_count = current_count.get('count')
+                count_map[row[index]] = {"count": (current_count + 1), "fuzzy": False}
+
+        # if the fuzzy threshold is set to 100% match, we can skip this
+        # O Notation murdering step
+        if fuzzy_thresh < 100:
+            # Get a list of the keys in the count_map
+            # Then see if any are a fuzzy match
+            count_map = self.count_fuzzy_matches(count_map, fuzzy_thresh, remove_numbers)
+
+        alert_list = sorted(count_map.items(), 
```

Comment:
It looks like we miss out on the sorted response with this change, is there a way to get that back?

Reply:
believe so - lemme fuss around; I think I ditched this early on but maybe not for the right reason